loops.js

```diff
@@ -16,9 +18,23 @@
 // loop...
 // loop...
 // klaar!
- // ==========================================
+// ==========================================
 
+for (let i = 0; i < 5; i++) {
+    if (i < 4) {
+        console.log("loop...");
+    }
+    if (i === 4) {
+        console.log("klaar!");
+    }
+}
 
+// of  if (i === 4) {
+//         console.log("klaar!");
+//     }
+//     else {
+//         console.log("loop...");
+//     }
```

Comment:
Die met else is beter, leesbaarheid is beter, let op je brackets, else mag na de sluitende bracket op 34.